src/Runtime/ExecutionSession.hpp

```diff
@@ -59,6 +72,14 @@ class ExecutionSession {
   ~ExecutionSession();
 
 protected:
+  // Error reporting processing when throwing runtime errors. Set errno as
+  // appropriate.
+  std::string reportLibraryOpeningError(const std::string &libraryName) const;
```

Comment:
Can these be private instead?

Reply:
I made them private at this time, there is a derived class that might use them in the future (python interface), but does not at this time.